Formula/e/erlang.rb

```diff
@@ -35,14 +34,32 @@ class Erlang < Formula
   depends_on "wxwidgets" # for GUI apps like observer
 
   uses_from_macos "libxslt" => :build
+  uses_from_macos "ncurses"
+  uses_from_macos "zlib"
+
+  on_macos do
+    depends_on "coreutils" # sha(1|256)sum used for ex_doc download validation
+  end
```

Comment:
Is this still needed after we've switched to downloading `ex_doc` using a `resource` block?

Reply:
I don't think so; this is only the internal validation which Homebrew is probably already doing.

Reply:
I've since removed it, squashed the changes and force-pushed.